README.md

```diff
@@ -18,15 +18,41 @@ Or with bundler,
 gem "hubspot-ruby"
 ```
 
-## Authentication with an API key
+## Getting Started
+This library can be configured to use OAuth or an API key. To find the
+appropriate values for either approach, please visit the [HubSpot API
+Authentication docs].
+
+Below is a complete list of configuration options with the default values:
+```ruby
+Hubspot.configure({
+  hapikey: <HAPIKEY>,
+  base_url: "https://api.hubapi.com",
+  portal_id: <PORTAL_ID>,
+  logger: Logger.new(nil),
```

Comment:
I'm not certain why this defaults to `nil`. I think we want this to default to `Logger.new(STDOUT)`.

Reply:
I think defaulting to nil means that it isn't logging which may be a reasonable default if this is noisy.